app/models/user.rb

```diff
@@ -11,6 +11,7 @@ class User < ApplicationRecord
   has_one :admin_profile, dependent: :destroy
   has_one :early_career_teacher_profile, dependent: :destroy
   has_one :mentor_profile, dependent: :destroy
+  has_many :npq_profiles, dependent: :destroy
```

Comment:
differs from other profile types as this one is not singleton

Reply:
How come?

Reply:
a user can register for multiple NPQs so we need this as a has many